### DISCUSSION TITLE
Multi Language for user switch support?

### DISCUSSION THREAD

**Discussion by 8MiYile** [Zrips/CMI#7601] [Feature Requests]
Can CMI read the language of the client to display the corresponding language?
For example, if Player A is an English user and Player B is a Chinese user, they can both use the corresponding language output of the client when using CMI

**Comment by CloudeLecaw** [comment]
and this would force us poor admins to check all language files to be correct @.@

**Reply by 8MiYile** [reply]
As soon as this solution can be supported, someone will come and adapt their language, just like the translation for the game Minecraft
